test/expression/switch.stmt

```diff
@@ -28,7 +28,7 @@ e = switch (c) {
 e = switch (c) {
   first => a,
   second =>
-      veryLongExpression + thatSplits,
+    veryLongExpression + thatSplits,
```

Comment:
This is the style change. By indenting bodies +2 instead of +4, it makes it more natural to indent the `when` clause +4 when it splits. If we indent the body +4, then you have to do like +8 for the `when` clause which looks pretty extreme.